packages/terra-clinical-result/src/common/other/_EnteredInError.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import styles from './OtherTemplates.module.scss';
+
+const cx = classNames.bind(styles);
+
+const EnteredInError = (props) => {
+  const {
+    ...customProps
+  } = props;
+
+  const templateClassnames = cx([
+    'entered-in-error',
+  ]);
+
+  return (
+    // TODO add i18n - in progress
```

Comment:
Where are we with i18n/

Reply:
Added here: https://github.com/cerner/terra-clinical/pull/574/commits/19fa7ce1fc26da485a72dcea5630562257d06e43